packages/origin-ui-core/src/components/devices/deviceMenuCreator.tsx

```diff
@@ -56,6 +57,13 @@ export const deviceMenuCreator = (user: IUser, t: TFunction): IDeviceMenuItem[]
         user?.organization && isRole(user, Role.OrganizationDeviceManager, Role.OrganizationAdmin);
 
     return [
+        {
+            key: 'import',
+            label: t('navigation.devices.import'),
+            component: ImportDevice,
+            features: [OriginFeature.Devices],
```

Comment:
This menu item, for now should be hidden under specific feature flag, like `OriginFeature.DeviceImport`. 
And this feature should be disabled in `OriginConfigurationContext` for now.

Reply:
done